scripts/expected_doxygen_warnings.txt

```diff
@@ -1,70 +1,72 @@
+/home/runner/work/cbmc/cbmc/src/cpp/cpp_typecheck.h:275: warning: documented empty return type of cpp_typecheckt::full_member_initialization
+/home/runner/work/cbmc/cbmc/src/goto-instrument/dot.cpp:59: warning: documented empty return type of dott::find_next
+/home/runner/work/cbmc/cbmc/src/goto-instrument/dot.cpp:48: warning: documented empty return type of dott::write_dot_subgraph
+/home/runner/work/cbmc/cbmc/src/goto-instrument/dot.cpp:54: warning: documented empty return type of dott::write_edge
+/home/runner/work/cbmc/cbmc/src/goto-cc/goto_cc_mode.h:28: warning: documented empty return type of  goto_cc_modet::usage_error
+/home/runner/work/cbmc/cbmc/src/goto-symex/goto_symex.h:693: warning: argument 'state' from the argument list of goto_symext::symex_cpp_new has multiple @param documentation sections
+/home/runner/work/cbmc/cbmc/src/goto-symex/goto_symex.h:693: warning: argument 'lhs' from the argument list of goto_symext::symex_cpp_new has multiple @param documentation sections
+/home/runner/work/cbmc/cbmc/src/goto-symex/goto_symex.h:693: warning: argument 'code' from the argument list of goto_symext::symex_cpp_new has multiple @param documentation sections
+/home/runner/work/cbmc/cbmc/jbmc/src/java_bytecode/java_bytecode_language.h:194: warning: documented empty return type of  java_bytecode_languaget::methods_provided
+/home/runner/work/cbmc/cbmc/src/analyses/reaching_definitions.cpp:675: warning: found </tt> tag without matching <tt>
+/home/runner/work/cbmc/cbmc/src/analyses/reaching_definitions.cpp:676: warning: found </tt> tag without matching <tt>
+/home/runner/work/cbmc/cbmc/src/analyses/reaching_definitions.cpp:677: warning: found </tt> tag without matching <tt>
+/home/runner/work/cbmc/cbmc/src/analyses/reaching_definitions.cpp:679: warning: found </tt> tag without matching <tt>
+/home/runner/work/cbmc/cbmc/src/analyses/reaching_definitions.h:221: warning: The following parameters of rd_range_domaint::merge(const rd_range_domaint &other, locationt from, locationt to) is not documented:
   parameter 'other'
-/cbmc/src/solvers/smt2/smt2_conv.h:113: warning: documented empty return type of  smt2_convt::convert_rounding_mode_FPA
-/cbmc/src/goto-symex/symex_slice_class.h:25: warning: documented empty return type of  symex_slicet::collect_open_variables
+/home/runner/work/cbmc/cbmc/src/solvers/smt2/smt2_conv.h:113: warning: documented empty return type of smt2_convt::convert_rounding_mode_FPA
+/home/runner/work/cbmc/cbmc/src/goto-symex/symex_slice_class.h:25: warning: documented empty return type of symex_slicet::collect_open_variables
+/home/runner/work/cbmc/cbmc/src/goto-instrument/cover_basic_blocks.h:122: warning: argument 'block.' of command @param is not found in the argument list of cover_basic_blockst::add_block_lines(cover_basic_blockst::block_infot &block, const goto_programt::instructiont &instruction)
```

Comment:
This warning does not seem to appear in the old version. This isn't a problem in itself, but the commit is claiming its only changing path and line numbers.